app/plex_auto_collections.py

```diff
@@ -35,86 +36,120 @@ def update_from_config(config_path, plex, headless=False, no_meta=False, no_imag
-                    if m == "tmdb_id":
-                        met = "tmdb_list"
-                        if not tmdb_id:      tmdb_id = v
-                        v = "https://www.themoviedb.org/collection/" + v
-
-                    check = True
-                    if (m == "tmdb_id" or m == "tmdb_list") and not TMDB.valid:
-                            print("| Config Error: {} skipped. tmdb incorrectly configured".format(m))
-                            check = False
-
-                    if check:
-                        try:                            missing = add_to_collection(config_path, plex, met, v, c, subfilters)
-                        except UnboundLocalError:       missing = add_to_collection(config_path, plex, met, v, c)               # No sub-filters
-                        except (KeyError, ValueError) as e:
-                            print(e)
-                            missing = False
-                        if missing:
-                            if libtype == "movie":
-                                method_name = "IMDb" if "imdb" in m else "Trakt" if "trakt" in m else "TMDb"
-                                print("| {} missing movies from {} List: {}".format(len(missing), method_name, v))
-                                if Radarr.valid:
-                                    radarr = Radarr(config_path)
-                                    if radarr.add_movie:
-                                        print("| Adding missing movies to Radarr")
-                                        add_to_radarr(config_path, missing)
-                                    elif not headless and radarr.add_movie == None and input("| Add missing movies to Radarr? (y/n): ").upper() == "Y":
-                                        add_to_radarr(config_path, missing)
-                            elif libtype == "show":
-                                method_name = "Trakt" if "trakt" in m else "TMDb"
-                                print("| {} missing shows from {} List: {}".format(len(missing), method_name, v))
-                                # if not skip_sonarr:
-                                #     if input("Add missing shows to Sonarr? (y/n): ").upper() == "Y":
-                                #         add_to_radarr(missing_shows)
+                    try:                                    missing, map = add_to_collection(config_path, plex, m, v, c, map, subfilters)
+                    except UnboundLocalError:               missing, map = add_to_collection(config_path, plex, m, v, c, map)               # No sub-filters
+                    except (KeyError, ValueError) as e:
+                        print(e)
+                        missing = False
+                    if missing:
+                        if libtype == "movie":
+                            method_name = "IMDb" if "imdb" in m else "Trakt" if "trakt" in m else "TMDb"
+                            if m == "tmdb_list" or m == "tmdb_list" or m == "imdb_list":
```

Comment:
Should this be:
`if m == "tmdb_list" or m == "trakt_list" or m == "imdb_list":`

Reply:
nope should be trakt I updated it good find